code/parachain/frame/staking-rewards/src/lib.rs

```diff
@@ -391,6 +399,10 @@ pub mod pallet {
 		type TreasuryAccount: Get<Self::AccountId>;
 
 		type ExistentialDeposits: GetByKey<Self::AssetId, Self::Balance>;
+
+		/// The default existential deposit for newly created share tokens
+		#[pallet::constant]
+		type ShareAssetExistentialDeposit: Get<Self::Balance>;
```

Comment:
This can likely be set to zero unconditionally, since the share asset is fully managed. Removing this config parameter and having an ED of zero also removes any possibility of share asset transfer issues; i.e if the ED is zero, then we don't have to worry about min stake > share token ED on pool creation.

Reply:
c2cdf764900050823838e4f8b70d4dbd2f723030